woocommerce/woocommerce-gateway-stripe
# Fix inconsistent payment processing error messages

# Changes proposed in this Pull Request:

Fixes #1187

- Change all occurrences of "card's" to "card" to simplify the messaging.

# Testing instructions

N/A

These strings should never be displayed to a customer because they're handled by Stripe's Card Element or we display the error coming directly from Stripe. The strings are only used to create order notes for the merchant.

You can verify this by trying to checkout with `4000 0000 0000 0127` which is supposed to return `incorrect_cvc`, but instead of the customer seeing the string we provide:

> The card security code is incorrect.

The customer will instead see

> Your card's security code is incorrect.

-------------------
